content/index.html.haml

```diff
@@ -36,12 +36,13 @@ homepage: true
 = partial('downloadbanner.html.haml')
 
 = partial('projectjumbotron.html.haml',
-  :href => expand_link('projects/jenkins-x'),
-  :background => '#0a1922',
-  :title => 'Sail into Jenkins X',
-  :intro => "Jenkins X is a CI/CD solution for modern cloud applications on Kubernetes.",
-  :image => expand_link('images/jenkins-x/logo.svg'),
-  :call_to_action => {:text => 'Get started', :href => expand_link('projects/jenkins-x')})
+  :href => 'https://www.cloudbees.com/devops-world/san-francisco',
+  :background => '#000000',
+  :title => 'Join us in San Francisco',
+  :intro => "DevOps World - Jenkins World is the largest gathering of Jenkins users in the world, including Jenkins experts, continuous delivery practitioneers, and companies offering complementary technologies to Jenkins. Join us in San Francisco, CA, from September 16-19.",
```

Comment:
This is the kind of text which I'm not very fond of, this is pretty much verbatim marketing text.

What I had in mind when we chatted about this was something more along the lines of "Join the Jenkins project for a contributor summit, and whole track of Jenkins related talks at DevOps World - Jenkins World on $DATESTAMP. in San Francisco"